public/js/types.js

```diff
@@ -7,7 +7,7 @@ const Source = t.struct({
   url: t.union([t.String, t.Nil]),
 
   // Internal for Firefox for now
-  actor: t.String
+  actor: t.maybe(t.String)
```

Comment:
Actually Jason, you were right, the new API automatically works because it just takes the `sourceId` in functions and already passes it as `actor`. I just pushed a small commit to remove this field.